CONTRIBUTING.md

```diff
@@ -1,112 +1,113 @@
 
 9. By default locally, you will be logged with `admin` access. If you want to change it, need to set `RSSCHOOL_AUTH_DEV_ADMIN` to `false` in `nestjs/.env` file
 
-**IMPORTANT:** Never commit changes to `.env` file
+    **IMPORTANT:** Never commit changes to `.env` file
 
-10. Do not forget to write [Jest](https://facebook.github.io/jest/) tests for your feature following [Specs Styleguide](#specs-styleguide)
+10. Do not forget to write unit-tests for your feature following [Unit-Tests Style Guide](#unit-tests-style-guide). We use [Jest](https://facebook.github.io/jest/) for unit-tests.
 
-11. Write end-to-end tests for your feature if applicable. Please see `client/specs` directory for more information. We use [Playwright](https://playwright.dev/) for end-to-end tests. You can run them using `npm run test:e2e` command and they suposed to work against test database snapshot.
+11. Write end-to-end tests for your feature if applicable. Please see `client/specs` directory for more information. We use [Playwright](https://playwright.dev/) for end-to-end tests. You can run them using `npm run test:e2e` command and they supposed to work against test database snapshot.
 
 12. Make sure tests, lints pass and code formatted properly (they'll be run on a git pre-commit hook too)
 
-```command-line
-$ npm test
-$ npm run lint
-$ npm run pretty
-```
+    ```command-line
+    npm test
+    npm run lint
+    npm run pretty
+    ```
 
-14. Commit your changes using a descriptive commit message that follows our [commit message conventions](#git-commit-messages)
+13. Commit your changes using a descriptive commit message that follows our [Commit Message Conventions](#git-commit-messages)
 
-```command-line
-$ git commit -m "feat: implement feature X"
-```
+    ```command-line
+    git commit -m "feat: implement feature X"
+    ```
 
-15. Push your branch to GitHub:
+14. Push your branch to GitHub:
 
-```command-line
-$ git push origin feature-x
-```
-
-16. Create a pull request. We support "feature branch" deployments. If you want to deploy your pull request, please add `deploy` label during creation.
+    ```command-line
+    git push origin feature-x
+    ```
 
+15. Create a pull request. We support "feature branch" deployments. If you want to deploy your pull request, please add `deploy` label during creation.
 
-### API client generation
+## API client generation
 
-We use [OpenAPI Generator](https://openapi-generator.tech/) to generate API client for `nestjs` API. Here are steps how to do it:
+We use [OpenAPI Generator](https://openapi-generator.tech/) to generate API client for `NestJS` API. Here are steps how to do it:
 
-- Make sure database is runing locally
+- Make sure that [Java](https://www.java.com/) is installed
```

Comment:
According to [OpenAPI Generator docs](https://openapi-generator.tech/docs/installation) there are options of running OpenAPI generator without Java installed (e.g. using Docker). Please consider to rephrase it somehow

Reply:
fixed